### PR TITLE
Gurindam keyboard 

### PR DESCRIPTION
**Pull request by ibnuda**
## Description
a cheap clone of arisu keyboard with promicro. it was a group buy for a regional community.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/gurindam/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+![Gurindam](https://cdn.discordapp.com/attachments/406032987243806721/922514068265369611/IMG_20211220_222843.jpg)
+
+A keyboard inspired by Arisu which in turn by Lyn's EM7 and TGR Alice.
+
+* Keyboard Maintainer: [Ibnu D. Aji](https://github.com/ibnuda)
+* Hardware Supported: Promicro
+* Hardware Availability: [PCB & Stacked Acrylic Case](https://github.com/ibnuda/gurindam) & private group-buy.
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/gurindam/readme.md:11]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/gurindam/rules.mk [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```
Comment (drashna) [inline keyboards/gurindam/rules.mk:15]:
```suggestion
```

File: keyboards/gurindam/config.h [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+#define MATRIX_ROW_PINS { B5, B4, E6, D7, C6, D4, D0, D1, D2}
+#define MATRIX_COL_PINS { F6, F5, F4, F7, B1, B3, B2, B6 }
+#define UNUSED_PINS
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+
+#define RGB_DI_PIN D3
+#define RGBLED_NUM 10
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/gurindam/config.h:52]:
```suggestion
```

File: keyboards/gurindam/config.h [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+// /*== or choose animations ==*/
+//   #define RGBLIGHT_EFFECT_BREATHING
+//   #define RGBLIGHT_EFFECT_RAINBOW_MOOD
+//   #define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+//   #define RGBLIGHT_EFFECT_SNAKE
+//   #define RGBLIGHT_EFFECT_KNIGHT
+//   #define RGBLIGHT_EFFECT_CHRISTMAS
+//   #define RGBLIGHT_EFFECT_STATIC_GRADIENT
+//   #define RGBLIGHT_EFFECT_RGB_TEST
+//   #define RGBLIGHT_EFFECT_ALTERNATING
```
Comment (drashna) [inline keyboards/gurindam/config.h:77]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/gurindam/rules.mk [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # USB Nkey Rollover
```
Comment (fauxpark) [inline keyboards/gurindam/rules.mk:15]:
```suggestion
NKRO_ENABLE = yes           # Enable N-Key Rollover
```

File: keyboards/gurindam/readme.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# gurindam
+
+![Gurindam](https://cdn.discordapp.com/attachments/406032987243806721/922514068265369611/IMG_20211220_222843.jpg)
```
Comment (fauxpark) [inline keyboards/gurindam/readme.md:3]: Please reupload this on imgur.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/gurindam/readme.md (hunk 6 above)
Comment (ibnuda) [reply keyboards/gurindam/readme.md]: i've uploaded the image to the github. hope it's ok.

**Review by fauxpark — APPROVED** [review]

**Pull request merged**